sdk/src/wallet/core/mod.rs

```diff
@@ -94,14 +94,15 @@ where
         Ok(self.accounts.read().await.clone())
     }
 
-    /// Get all account aliases
-    pub async fn get_account_aliases(&self) -> crate::wallet::Result<Vec<String>> {
+    /// Get all account identifiers
+    pub async fn get_account_ids(&self) -> crate::wallet::Result<Vec<(u32, String)>> {
```

Comment:
Breaking change? Revert

Reply:
shiiiiit

Reply:
You can add another method if really needed but don't touch this one

Reply:
okay, I am going to add another fn that returns the indexes for the time being until we want to make a breaking release.

Reply:
nah, this has problems, too ... I am going to have to use `get_accounts`.